src/yarn-frameworklauncher/deploy/yarn-frameworklauncher.yaml.template

```diff
@@ -62,6 +62,13 @@ spec:
           value: {{ clusterinfo[ 'frameworklauncher' ][ 'frameworklauncher_vip' ] }}
         - name: FRAMEWORKLAUNCHER_PORT
           value: "{{ clusterinfo[ 'frameworklauncher' ][ 'frameworklauncher_port' ] }}"
+        - name: LAUNCHER_OPTS
+          value: "-server -Xmx3072m -Djute.maxbuffer=49107800"
+        resources:
+          limits:
+            memory: "4Gi"
+          requests:
```

Comment:
Why need to set requests less than limits?
Suggest to remove requests to make it as Guaranteed Pod

Reply:
Will need to set both 'cpu' and 'memory' to make it 'Guaranteed'.
For now we haven't figure out a reasonable cpu limits, so most services are 'Burstable'.
And Launcher depended on 'hadoop' services, which are  'Guaranteed', so that only making launcher 'Guaranteed' won't bring benefit.